compilers/concrete-compiler/compiler/lib/ClientLib/ClientLib.cpp

```diff
@@ -130,5 +130,81 @@ Result<ClientCircuit> ClientProgram::getClientCircuit(std::string circuitName) {
                      "`");
 }
 
+Result<TfhersFheIntDescription>
+getTfhersFheUint8Description(llvm::ArrayRef<uint8_t> serializedFheUint8) {
+  auto fheUintDesc = concrete_cpu_tfhers_uint8_description(
+      serializedFheUint8.data(), serializedFheUint8.size());
+  if (fheUintDesc.width == 0)
+    return StringError("couldn't get fheuint info");
+  return fheUintDesc;
+}
+
+Result<TransportValue>
+importTfhersFheUint8(llvm::ArrayRef<uint8_t> serializedFheUint8,
+                     uint32_t encryptionKeyId, double encryptionVariance) {
+  auto fheUintInfoOrError = getTfhersFheUint8Description(serializedFheUint8);
```

Comment:
use OUTCOME macros

```
  OUTCOME_TRY(auto fheUintDesc,
              getTfhersFheUint8Description(serializedFheUint8));
```

Reply:
this was removed completely as we no longer need to get description, but build it from tfhers params